docroot/themes/humsci/humsci_basic/templates/menus/macros/secondary-nav-menu.twig

```diff
@@ -35,11 +35,11 @@
           acts as the button to toggle the nested navigation.
         #}
         {% set link_attributes = link_attributes.addClass('hb-secondary-toggler') %}
-        {% set randomInt = random(0, 100) %}
-        <a{{ link_attributes }} id="{{ item.title|replace({' ': ''}) }}{{ menu_level }}{{ randomInt }}" aria-expanded="true" role="button">{{ item.title }}</a>
+        {% set unique_id = item.title|clean_class|clean_unique_id %}
```

Comment:
Sorry, a final change...  I was looking how `clean_unique_id` works and I'm pretty sure you don't need `clean_class`, because `clean_unique_id` also does the clean.

Please check and update.